### ISSUE TITLE
Feature request : Clear data for currently visited site only

### ISSUE THREAD

**Issue by mwamp** [dessant/clear-browsing-data#36]
Thanks for your work and this extension

I am not interested in a global data wipe, I believe it is only a few clicks away in firefox if I need it occasionally. For a single website though, a one click option is very useful instead of calling the console/storage panel and wipe everything by hand.

Would you consider adding a option that wipes data only for the website of the current tab?

**Comment by pythoninthegrass** [comment]
Came here to say the same. Excellent add-on regardless. Saves me a few clicks for a problem child website that I'm constantly running in private mode due to expired session cookies.

@dessant Let me know if you need help testing this feature. Happy to debug on both Firefox and Chrome.

**Comment by i486** [comment]
I feel like this shouldn't be too hard to implement considering Firefox enabled Total Cookie Protection globally and its own "Menu > History > Forget About This Site" feature essentially does this and remove everything for the selected site. All Clear Browsing Data has to add is internally use "Forget About This Site" or just simply remove the entire "jar" for the currently open site. 

I am not a developer. Feel free to correct me if i am wrong.

**Comment by i486** [comment]
This is the only other alternative that can clear current site browser data with one-click
https://addons.mozilla.org/en-US/firefox/addon/forget-about-this-site/

Sadly, it's not updated and it doesn't support partitioned cookies (Total Cookie Protection).
This means when a user visits https://examplesite.com/ and the site creates cookies for examplesite.com and example**tracker**.com. 

If used, it will only delete examplesite.com cookie but not example**tracker**.com, leaving user vulnerable to tracking at later date by examplesite.com

**Comment by objecttothis** [comment]
I too would like the option to specifically clear browsing data related to the current active tab.

**Comment by panckreous** [comment]
yes please. in the mean time, what are the best ways to do this?

**Comment by peterkassner** [comment]
I agree, and I'd pay a small license fee for the privilege.

**Comment by ldorigo** [comment]
This would be a boon for developers, currently I need something like 15 clicks in the devtools - clear cookies, clear session cookies, clear local storage, clear session storage, clear indexed db, ........

**Comment by 3dot141** [comment]
https://github.com/dessant/clear-browsing-data/pull/61
i create pr for it, but can't merged due to this strategy
